integration-test/common/src/test/java/com/tencent/rss/test/IntegrationTestBase.java

```diff
@@ -97,7 +97,7 @@ protected static void createCoordinatorServer(CoordinatorConf coordinatorConf) t
   }
 
   protected static void createShuffleServer(ShuffleServerConf serverConf) throws Exception {
-    shuffleServers.add(new ShuffleServer(serverConf));
+    shuffleServers.add(new MockedShuffleServer(serverConf));
```

Comment:
add new method that createMockShuffleServer.

Reply:
ok

Reply:
Added